ui/ui-guidelines.md

```diff
@@ -1,17 +1,60 @@
 # UI Guidelines
 
-## Background
-
-This lays out the current state of how the frontend code is architected, the best
-practices to follow when adding new code to the frontend and the considerations
+This document lays out the current state of how the frontend code is architected, the best
+practices to follow when adding new code to the frontend, and the considerations
 for the future growth of the frontend. This is meant to be a living document that
-will evolve as the UI grows. There are lots of exceptions to every rule laid out
+will evolve as the UI grows.
+
+There are lots of exceptions to every rule laid out
 here as UI code can vary significantly from feature to feature.
 
-## Elements of the UI
+> ⚠️ You may find inconsistencies with some of the best practices in older code that was written before these guidelines were updated. New code should follow these guidelines. We are slowly working on bringing older code in line with the best practices.
+
+## Notable tooling choices
+
+Below is a list of notable tooling that is used throughout the Perses UI.
```

Comment:
Am I missing anything notable in this list? I'm not trying to reproduce the package.json here, but am trying to make it easy to figure out the big stuff.

Reply:
not sure if we need to include this, but we use [react-colorful](https://github.com/omgovich/react-colorful) for color picker